src/extensions/endpoints/credits-timeline/src/index.ts

```diff
@@ -40,7 +40,7 @@ export default defineEndpoint((router, context) => {
 
 			const query = value.query as unknown as {offset: number, limit: number};
 
-			const changes = await database.with('rows', database.unionAll([
+			const changesSql = database.unionAll([
 				database('gp_credits_additions')
 					.join('directus_users', 'gp_credits_additions.github_id', 'directus_users.external_identifier')
 					.where('directus_users.id', value.accountability!.user!)
```

Comment:
It would be good to also extend this to support the new impersonation feature: the client can pass a username (or empty) to show all data), the backend accepts it if the real user has an admin role.

Reply:
Actually, I'll open a separate issue for this.